index.html

```diff
@@ -18,7 +18,7 @@
         
           <h1 id="title">Survey Form</h1>
       
-          <center><p id="description"><i>Please take a moment to fill our survey. Thank you</i></p></center>
```

Comment:
id of elements are supposed to be unique in the dom. but there are 2 dom elements with the same id description it would be nice if you could change one of them.